nemo/collections/nlp/models/language_modeling/megatron_gpt_model.py

```diff
@@ -100,8 +99,7 @@ def __init__(self, cfg: DictConfig, trainer: Trainer):
             fp16_lm_cross_entropy=cfg.get('fp16_lm_cross_entropy', False),
             use_cpu_initialization=cfg.get('use_cpu_initialization', False),
             hidden_dropout=cfg.get('hidden_dropout', 0.1),
-            fused_fp16=cfg.get('fused_fp16', False),
-            fused_bf16=cfg.get('fused_bf16', False),
+            precision=cfg.get('precision', 16),
```

Comment:
Same comment here - maybe we want to add precision to the model config via ${trainer.precision}?

Reply:
Yup, that's right. I'll make the change.

Reply:
I ended up not being able to use interpolation as the ${trainer.precision} could not be found when PTL saves the hparams. I added a comment explaining why we add the precision to the model config in the training script.